src/App.tsx

```diff
@@ -43,15 +43,16 @@ function App() {
   return (
     <div className="m-[15%]">
       <div className="grid grid-cols-[repeat(auto-fill,minmax(200px,1fr))]">
-        <Clock
-          name="Quartz"
-          tz="America/Los_Angeles"
-          twitter="piezo_electric"
-        />
-        <Clock name="Ava" tz="America/Los_Angeles" twitter="avaaa_42" />
+        <Clock name="Ava" tz="America/New_York" twitter="avapsilver" />
+        <Clock name="Vivi" tz="America/New_York" twitter="vivi_is_tired" />
+        <Clock name="Philo" tz="America/New_York" twitter="flaming_spork" />
+        <Clock name="Maxine" tz="America/New_York" twitter="prsmaticwolf" />
+        <Clock name="Tris" tz="America/New_York" />
+        <Clock name="Liz" tz="America/Los_Angeles" twitter="selectric401" />
+        <Clock name="Mia" tz="America/New_York" twitter="nkizz11" />
+        <Clock name="Ezgi" tz="America/New_York" twitter="ezgiburglar69" />
         <Clock name="Kay" tz="America/New_York" twitter="ffoorrggg" />
         <Clock name="Ula" tz="America/New_York" twitter="oh_jeez_bees" />
```

Comment:
these oomfs are cool but given that they're all in my TZ and there's a limited amount of screen space to maintain the aesthetic, suggest remove

Reply:
:cat_salute: